src/lib.rs

```diff
@@ -23,7 +23,7 @@
     clippy::unnecessary_unwrap,
     clippy::pedantic
 )]
-
+#![warn(clippy::option_if_let_else)]
```

Comment:
isn't that included in the defaults?

Reply:
default is `deny`. this lets it get through, only emitting a warning.